src/Microsoft.Data.Sqlite/SqliteCommand.cs

```diff
@@ -104,6 +106,8 @@ public new virtual SqliteDataReader ExecuteReader(CommandBehavior behavior)
                         : Strings.TransactionConnectionMismatch);
             }
 
+            NativeMethods.sqlite3_busy_timeout(Connection.DbHandle, CommandTimeout * 1000);
```

Comment:
Maybe add a TODO to only do this if it's changed since the last time we called it on this open connection.